src/components/search-criteria.js

```diff
@@ -2,18 +2,29 @@ import React, { Component } from 'react';
 
 export default class SearchCriteria extends Component {
   render() {
-    const { keyword, address, region, category, numOfResults } = this.props;
+    /* Note that numOfResults is currently ignored, in favour of
+    numOfResultsDisplayed. This reflects the default limit of 50 search provider
+    results displayed in index.js. 
+
+    If pagination of results is added in future numOfResults will become useful
+    again to show the total number of results found across all pages, for
+    example "Showing ${numOfResultsDisplayed} of {numOfResults} results". */
+    // eslint-disable-next-line no-unused-vars
+    const { keyword, address, region, category, numOfResults, numOfResultsDisplayed } = this.props;
```

Comment:
@lamorrison @JacOng17 Looks like numOfResults isn't being used?

Reply:
Yup, that's right, we just left it there b'cos might need it in future. Added `eslint-disable-next-line no-unused-vars` to ignore it.

Reply:
@lamorrison @JacOng17 Jacob explained we are leaving this variable here for pagination use later on.